ERCS/erc-draft_pm_capability.md

```diff
@@ -0,0 +1,285 @@
+---
```

Comment:
```suggestion
---
eip: 7677
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.